Vagrantfile

```diff
@@ -56,11 +56,45 @@ def vm_memory
   $vb_memory.nil? ? $vm_memory : $vb_memory
 end
 
+PERSISTENT_DISK_DIR = File.join(File.expand_path('~'), '.bricks-data')
+PERSISTENT_DISK = File.join(PERSISTENT_DISK_DIR, 'persistent_data.vdi')
```

Comment:
so virtual box auto creates this? Or, do we need to create this as part of the setup process?

Reply:
This is auto created on vagrant up if it does not exist